sphinx/source/docs/user_guide/tools.rst

```diff
@@ -486,32 +484,29 @@ in data or screen space. These special fields are listed here:
 Field names that begin with ``@`` are associated with columns in a
 ``ColumnDataSource``. For instance, the field name ``"@price"`` will display
 values from the ``"price"`` column whenever a hover is triggered. If the hover
-is for the 17th glyph, then the hover tooltip will correspondingly display
-the 17th price value.
+is for the 17th glyph, then the hover tooltip will display the 17th price value.
```

Comment:
```suggestion
is for the 17th glyph instance, then the hover tooltip will display the 17th price value.
```

This has always been a bit imprecise, but as long as we are editing this file, I think the above is a slight improvements. 

@tcmetzger this is actually maybe something good for your Glossary. We have never nailed down any concrete consistent terminology to distinguish "glyph" the object that can draw a bunch of vectorized visuals, and "glyph" one of the individual visuals that gets draw. But this is an important thing to be able to distinguish.

Reply:
That definitely helps to make it clearer!